Source/Widgets/Timeline/Timeline.js

```diff
@@ -251,6 +253,11 @@ define([
         }
         //>>includeEnd('debug');
 
+        var timesValid = JulianDate.greaterThanOrEquals(startTime, Iso8601.MINIMUM_VALUE) && JulianDate.lessThanOrEquals(stopTime, Iso8601.MAXIMUM_VALUE);
```

Comment:
@emackey can confirm but I suspect we would want to throw in this case instead of a no-op, since a developer might call this code directly.  We should then do this check before our own calls to `zoomTo` instead so that we only no-op in those cases.